user/harec/patches/install-target.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Makefile b/Makefile
+index a7b4ed3..00c8728 100644
+--- a/Makefile
++++ b/Makefile
+@@ -67,6 +67,7 @@ check: $(BINOUT)/harec $(tests)
+ 	@./tests/run
+ 
+ install: $(BINOUT)/harec
++	install -d $(DESTDIR)$(BINDIR)
+ 	install -Dm755 $(BINOUT)/harec $(DESTDIR)$(BINDIR)/harec
```

Comment:
think upstream in this case would take a fix for this (regular install -d + install -m after etc)

Reply:
already sent